iFixit/react-commerce
# Fix navigation z-index

closes #1285 

## QA

1. Open [Vercel preview](https://react-commerce-git-fix-navigation-zindex-ifixit.vercel.app/products/joy-con-lanyard)
2. Verify that #1285 is fixed
